task/upload.go

```diff
@@ -503,15 +517,6 @@ func complementCatalystPipeline(tctx *TaskContext, assetSpec api.AssetSpec, call
 		assetSpec.Storage.IPFS = &ipfs
 	}
 
-	_, metadataPath, err := saveMetadataFile(tctx, tctx.outputOS, tctx.OutputAsset.PlaybackID, metadata)
```

Comment:
What is this metadata, don't we need it anymore?

Reply:
This is the file I brought up on discord, seems like we're good to remove it. I'll wait for @victorges to check the PR though before merging.